src/listItems.js

```diff
@@ -0,0 +1,75 @@
+import ApiData from './mealAPI';
+import { setLocalStorage } from './loadStorage';
+import counter from './counterItem';
+
+export default async function displayList(likedRecipes) {
```

Comment:
- [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```